mgmt/rest/v1/rbody/body.go

```diff
@@ -32,6 +32,8 @@ import (
 	"github.com/urfave/negroni"
 )
 
+const deprecationInfo = "The client API V1 is being deprecated; switch to API v2. Find more information here: https://github.com/intelsdi-x/snap/issues/1637"
```

Comment:
I feel a bit confused. Issue #1637 suggests that https://github.com/intelsdi-x/snap-client-go is the new API v2. Regardless of the fact that it is still private and not accessible to end users, it doesn't seem to contain API v2 but the client built on top of this API. API v2 seems to be defined here: https://github.com/intelsdi-x/snap/tree/master/mgmt/rest/v2. Is that correct?